### DISCUSSION TITLE
I need a clean build please.

### DISCUSSION THREAD

**Discussion by OmegaBeam69** [ROM-builders/temporary#14582] [Q&A]
Hey sup? Brother I used to compile my kernel with trb_clang. Problem is the error I'm getting ca n only fixed by making a clean build. Can you please help me regarding that?

**Comment by Apon77** [comment]
Build link?

**Reply by OmegaBeam69** [reply]
@Apon77 https://cirrus-ci.com/task/5766271157927936

**Reply by Apon77** [reply]
I don't think clean build will solve the problem, its someting wrong with your kernel tree, first test #failed instruction in telegram
